define-and-ideate/README.md

```diff
@@ -0,0 +1,40 @@
+Phase 2 of Design Thinking is  **Define**.  In this phase, designers gather and analyze all of the information collected during the Empathize phase.  By identifying pain points and patterns in the information, a clear definition of the problem is crafted.
+
+Remember, as we learned in our discussion of the Empathize phase, never simply try to solve the problem as stated ([Don Norman, _The Design Of Everyday Things_](https://jnd.org/the-design-of-everyday-things-revised-and-expanded-edition/)).  That issue is usually just a symptom of the underlying problem. Instead, derive insights into the core problem by synthesizing the data collected in phase one, and use those insights to articulate a _meaningful_ and _actionable, human-centric_ problem statement.
+
+
+
+<details><summary><b>Revisit the Case Study</b></summary>
+<br>
+Consider this example scenario: a municipality has an aging infrastructure and an archaic system for managing road crews.  They receive numerous phone calls and verbal complaints reporting road issues. These issues are then recorded in spreadsheets or email threads before a road crew is dispatched.  Many issues are never acted upon and citizens often call with complaints that repairs have not been completed.
+
+The customer requests that a software development team provide a solution that ingests and consolidates data from multiple and varied existing data sources (email, spreadsheets) and then provides a daily report of new issues.
+
+The true solution however - the most meaningful result - might be to provide a free mobile app that allows citizens to easily report a road issue (pothole, debris, etc.) which loads a centralized queue for dispatchers.  The app allows the road crew to report real-time status of repairs.
+
+In this example, the customer's identification of the problem is influenced by the immediate pain point of having too much incoming data in too many formats.  The designer, however, delves to the root problem and envisions a more innovative solution.  One that not only addresses the unmanaged data, but provides real-time feedback to the citizen, dispatcher, and road crew so that nothing falls through the cracks and everyone is accountable for the condition of the roads.
+<hr>
+</details>
+
+Looking back at our case study, our customer likely presented a problem statement similar to the following:
+
+ &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_We need a system to ingest and consolidate road maintenance issues from multiple sources and in many formats._
+
+But is that the core problem, or just a symptom?  Wouldn't a more **_meaningful_** and **_actionable_**, **_human-centric_** problem statement be something like this:
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_The citizens in our town need to be able easily report road issues and follow the progress of repairs._
+
```

Comment:
"Ou city wants to improve the safety of our roads and make them more enjoyable to be used, as such anyone should be able to easily report road issues and follow the progress of repairs"